hello_world/main.py

```diff
@@ -0,0 +1,15 @@
+"""This is a simple hello world app"""
+
+___name___         = "Hello World"
+___license___      = "MIT"
+___dependencies___ = ["sleep", "app"]
+___categories___   = ["EMF"]
+
+import ugfx, os, time, sleep, app
+
+
+# initialize screen
+ugfx.init()
+ugfx.clear()
+
+ugfx.text(5, 5, "Hello World!", ugfx.BLACK)
```

Comment:
Do you want some `sleep.wfi()` here, so the Hello-world app doesn't exit immediately?

Reply:
Thank for letting me know. I have fixed it using https://github.com/emfcamp/Mk4-Apps/pull/77.
It contains a fix of imports and a close function